src/patternfly/patternfly-charts.scss

```diff
@@ -50,3 +51,97 @@ $pf-chart-color-black-200: $pf-color-black-400;
 $pf-chart-color-black-300: $pf-color-black-500;
 $pf-chart-color-black-400: $pf-color-black-600;
 $pf-chart-color-black-500: $pf-color-black-700;
+
+
+// typography
+$pf-chart-global--FontSize: $pf-global--FontSize--sm;
+
+// label
+$pf-chart-global-label--FontSize: $pf-chart-global--FontSize;
+$pf-chart-global-label--Padding: 10px;
+
+// Layout Props
+$pf-chart-global-layout--Padding: 50px;
+$pf-chart-global-layout--Height: 300px;
+$pf-chart-global-layout--Width: 450px;
+
+// Individual Charts
+
+// Area Chart
+$pf-chart-area--Opacity: .4;
+$pf-chart-area-stroke--Width: 2px;
+
+// Axis Chart
+$pf-chart-axis-axis-stroke--Width: 1px;
+$pf-chart-axis-axis-label--Padding: 100px;
+$pf-chart-axis--ticks--Width: 1px;  // this should be a var for size so not sure if width is appropriate
+
+// Bar Chart
+$pf-chart-bar--Width: 10px;
+
+// Box Plot Chart
+$pf-chart-boxplot-max--Padding: $pf-global--spacer--sm;
+$pf-chart-boxplot-max-stroke--Width: 1px;
+$pf-chart-boxplot-median--Padding: $pf-global--spacer--sm;
+$pf-chart-boxplot-median-stroke--Width: 1px;
+$pf-chart-boxplot-min--Padding: $pf-global--spacer--sm;
+$pf-chart-boxplot-min-stroke--Width: 1px;
+$pf-chart-boxplot-q1--Padding: $pf-global--spacer--sm;
+$pf-chart-boxplot-q3--Padding: $pf-global--spacer--sm;
+$pf-chart-boxplot-box--Width: 20px;
+
+// Bullet Chart
+$pf-chart-bullet--comparative-measure-line-warning--BorderColor: $pf-color-orange-300;
+$pf-chart-bullet--comparative-measure-line-warning--BorderWidth: 1px;
+$pf-chart-bullet--comparative-measure-line-error--BorderColor: $pf-color-red-100;
+$pf-chart-bullet--comparative-measure-line-error--BorderWidth: 1px;
+$pf-chart-bullet--zero-line--BorderColor: $pf-color-black-700;
+$pf-chart-bullet--zero-line--BorderWidth: 1px;
+$pf-chart-bullet--negative-primary-measure--BorderColor: $pf-color-red-100;
+$pf-chart-bullet--negative-primary-measure--BorderWidth: 2px;
+
+// Candlestick
+$pf-chart-candelstick-data-stroke--Width: 1px;
+
+// Chart
+$pf-chart-chart--BorderWidth: 1px;
```

Comment:
Should the "chart" variation of charts be defined globally? or is it actually a variation @dlabrecq

Reply:
Chart is a separate component. Basically it's a container which can be used with any chart to provide default functionality like a border, x and y axes, etc.

Reply:
FYI, the border is currently border: `1px solid`. Do we have `solid` defined anywhere?

Reply:
@dlabrecq when we usually define vars we wouldn't do one for solid, but this makes sense since its the only css defined for use in the victory charts

Reply:
They could also be separate vars and we concatinate them in the Victory theme?

Reply:
@mcoker what do you think? borders by default are solid right. Across the system I see alot of border widths without the 'solid'

Reply:
The only border I see in the Victory themes is for the Chart component -- not including borderWidth props. Keep in mind that these are SVGs, not CSS props.